web-app/js/portal/search/SearchBodyPanel.js

```diff
@@ -31,33 +32,100 @@ Portal.search.SearchBodyPanel = Ext.extend(Ext.Panel, {
             this._onResultsStoreLoad();
         }, this);
 
-        this.searchResultsPanel.pagingBar.on('beforechange', this._onResultsGridBbarBeforeChange, this);
+        this.on('render', function() {
+            var that = this;
+            this.getEl().down('div').down('.faceted-search-results').on('scroll', function (evt, c) {
+                that.handleScroll(evt, c);
+            });
+        }, this);
     },
 
     _onResultsStoreLoad: function() {
         if (this.resultsStore.getTotalCount() == 0) {
             this._displayNoResultsAlert();
         }
+        var that = this;
+        setTimeout(function () {
```

Comment:
If I'm using the scroll bar I can quite easily beat this timeout and then can't scroll anymore (until I go backwards a bit after 2 seconds have elapsed that is).  timeouts are bad

Reply:
I struggled without a timeout for quite a while. The issue is when the mouse is scrolled moderately fast, the mouse proximity to the bottom is within the trigger range even when new results load. So three or four paged requests can be made one after another before the mouse position is out of the trigger range.
I reason that someone racing through the results is not really using the Portal properly. Additionally scanning/ scrolling though the results  is so boring that filtering would become attractive.
The timeout could be reduced but the requests to Geoserver were taking a bit over a second at worst, so i settled on 2 seconds.

Reply:
Happy to merge if @kereid / @ccmoloney are OK with this. It works fine otherwise.

Reply:
I'm ok with it.